php/class-customize-snapshot-manager.php

```diff
@@ -1444,6 +627,74 @@ public function render_templates() {
 			<a href="javascript:void(0)" id="snapshot-expand-button" role="button" aria-controls="snapshot-schedule" aria-pressed="false" class="dashicons dashicons-edit"></a>
 		</script>
 
+		<script type="text/html" id="tmpl-snapshot-submit">
+			<button id="snapshot-submit" class="button button-primary">
+				{{ data.buttonText }}
+			</button>
+		</script>
+
+		<script type="text/html" id="tmpl-snapshot-dialog-error">
+			<div id="snapshot-dialog-error" title="{{ data.title }}">
+				<p>{{ data.message }}</p>
+			</div>
+		</script>
+
+		<script type="text/html" id="tmpl-snapshot-status-button">
+			<?php
+				$data = array(
+					'choices' => array(
+						'publish' => array(
+							'option_text' => esc_attr__( 'Publish' , 'customize-snapshots' ),
+							'alt_text' => esc_attr__( 'Published' , 'customize-snapshots' ),
+						),
+						'draft' => array(
+							'option_text' => esc_attr__( 'Save Draft' , 'customize-snapshots' ),
+							'alt_text' => esc_attr__( 'Draft' , 'customize-snapshots' ),
+						),
+						'future' => array(
+							'option_text' => esc_attr__( 'Schedule' , 'customize-snapshots' ),
+							'alt_text' => esc_attr__( 'Scheduled' , 'customize-snapshots' ),
+						),
+						'pending' => array(
+							'option_text' => esc_attr__( 'Save Pending' , 'customize-snapshots' ),
+							'alt_text' => esc_attr__( 'Pending' , 'customize-snapshots' ),
+						),
+					),
+					'selected' => 'publish',
+					'confirm_publish_text' => esc_attr__( 'Confirm Publish', 'customize-snapshots' ),
```

Comment:
Using `esc_attr__()` isn't appropriate in all of these cases. It should just be `__()`. The escaping will be handled at the moment where the string gets actually embedded into the document. The templating engine will escape them properly.